includes/REST_API/Stories_Base_Controller.php

```diff
@@ -42,21 +43,29 @@
  * Override the WP_REST_Posts_Controller class to add `post_content_filtered` to REST request.
  */
 class Stories_Base_Controller extends WP_REST_Posts_Controller {
-	use Decoder;
+	/**
+	 * Experiments instance.
+	 *
+	 * @var Experiments Experiments instance.
+	 */
+	private $experiments;
 
 	/**
 	 * Constructor.
 	 *
 	 * Override the namespace.
 	 *
-	 * @since 1.0.0
+	 * @since 1.1.0
 	 *
-	 * @param string $post_type Post type.
+	 * @param string      $post_type Post type.
+	 * @param Experiments $experiments Experiments instance.
 	 */
-	public function __construct( $post_type ) {
+	public function __construct( $post_type, Experiments $experiments ) {
 		parent::__construct( $post_type );
-		$this->namespace = 'web-stories/v1';
+		$this->namespace   = 'web-stories/v1';
+		$this->experiments = $experiments;
```

Comment:
```suggestion
		$this->experiments = new Experiments();
```

Reply:
I think it is cleaner if just do this. Meaning we don't have to do the `__construct` workaround.

Reply:
Can you elaborate? `$experiments` is passed down from the parent class. No need to instantiate it again.